src/monitor/monitor_bootstrap.c

```diff
@@ -95,7 +100,9 @@ int bootstrap_monitor_process(uid_t target_uid, gid_t target_gid)
                 sss_log(SSS_LOG_ALERT, "Failed to change uid:gid");
                 return 1;
             }
-        } else {
+        } else
+#endif
```

Comment:
Hi,

would you mind to add a `/* BUILD_CONF_SERVICE_USER_SUPPORT */` for completeness?

bye,
Sumit

Reply:
Added (and rebased)

Reply:
Thanks.